Cartfile.resolved

```diff
@@ -1,24 +1,24 @@
 github "wikimedia/AFNetworking" "22653dd7998b63cd168eda4b11be36b7b656ab96"
-github "wikimedia/BlocksKit" "440ecf74369ac75ab4500617a6b70641d9afa4d2"
-github "CocoaLumberjack/CocoaLumberjack" "2.4.0"
+github "wikimedia/BlocksKit" "bb5323c50f776988c2cea55c1d44be593266d22a"
+github "CocoaLumberjack/CocoaLumberjack" "3.0.0"
 github "Flipboard/FLAnimatedImage" "1.0.12"
 github "swisspol/GCDWebServer" "3.3.3"
 github "bitstadium/HockeySDK-iOS" "4.1.2"
-github "facebook/KVOController" "v1.1.0"
+github "facebook/KVOController" "v1.2.0"
 github "Mantle/Mantle" "2.1.0"
 github "SnapKit/Masonry" "v1.0.2"
-github "Quick/Nimble" "v4.1.0"
+github "Quick/Nimble" "v5.1.1"
 github "luisobo/Nocilla" "0.11.0"
 github "hamcrest/OCHamcrest" "v6.0.0"
 github "mxcl/PromiseKit" "3.5.2"
-github "Quick/Quick" "v0.9.3"
-github "wikimedia/SDWebImage" "bb49df83e72f2231a191e9477a85f0effe13430a"
+github "Quick/Quick" "v1.0.0"
 github "wikimedia/SSDataSources" "0fbc5784009c96c832f44ebefbe2d9d1501dac10"
 github "wikimedia/TUSafariActivity" "90e393a6555fc05de291a2d9036255f0172319fd"
 github "wikimedia/Tweaks" "52db8948eaa85e0a52870237cf0ae2d50a55289a"
 github "vtourraine/VTAcknowledgementsViewController" "1.2"
 github "wikimedia/hpple" "63951b4c33817321207bfa11ec850b22983cf03a"
-github "facebook/ios-snapshot-test-case" "2.1.3"
+github "facebook/ios-snapshot-test-case" "2.1.4"
 github "wikimedia/piwik-sdk-ios" "f3f5a7c65c43eb1628521f8fe3c19e9903da2c4d"
 github "NYTimes/NYTPhotoViewer" "1.2.0"
 github "jonreid/OCMockito" "v4.0.1"
+github "joewalsh/SDWebImage" "e904e013bcb4edd39f47fe6f50d2677524517b65"
```

Comment:
@coreyfloyd there is one currently and it wouldn't  let me make two - wanted to keep the old one around until after this release